crowbar_framework/app/models/node_object.rb

```diff
@@ -1321,12 +1321,57 @@ def boot_device!(device)
     boot_device(device) and save
   end
 
+  # TODO: Remove duplicate code and use a gem/git submodule/whatever...
+  # see barclamp-deployer/chef/cookbooks/barclamp/libraries/barclamp_library.rb
+  def unique_name_already_claimed_by(device)
+    claimed_disks = @node[:crowbar_wall][:claimed_disks].to_hash
+    claimed_disks.each do | claimed_name, v |
```

Comment:
@bkutil Done in https://github.com/toabctl/barclamp-crowbar/commit/b594de6e5845828a0b5bf4d90765f3ccfbfb349f